crates/refunder/src/main.rs

```diff
@@ -9,5 +9,6 @@ async fn main() {
     );
     shared::exit_process_on_panic::set_panic_hook();
     tracing::info!("running refunder with validated arguments:\n{}", args);
+    global_metrics::setup_metrics_registry(Some("refunder".into()), None);
```

Comment:
Note that the standard way to name metrics would have been `gp_v2_refunder`. I decided I could drop the prefix `gp_v2` here.

Reply:
This was important when we were under Gnosis infra to distinguish between the various projects. Agree that this can be dropped now.

I actually think we can drop the `gp_v2_*` prefix everywhere.